.github/workflows/push-release.yml

```diff
@@ -382,17 +382,25 @@ jobs:
           password: ${{ env.DOCKER_PASSWORD }}
       - name: Create manifest list and push
         run: |
-          docker buildx imagetools create -t ${{ env.IMAGE_AGENT }}:${{ env.TAG }} \
-            ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-amd64 \
-            ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-arm64 \
+          # docker manifest is used with windows images to maintain os.version in the manifest
+          docker manifest create ${{ env.IMAGE_AGENT }}:${{ env.TAG }} \
             ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-windows-2019 \
             ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-windows-2022
+
+          docker manifest push ${{ env.IMAGE_AGENT }}:${{ env.TAG }}
+
+          docker buildx imagetools create -t ${{ env.IMAGE_AGENT }}:${{ env.TAG }} \
```

Comment:
nit: it would be nice to have a comment highlighting that `imagetools create` pushes the resulting artefact to the target repo, as folks used to `manifest create` would expect that.